openmetadata-ui/src/main/resources/ui/src/pages/AddServicePage/AddServicePage.component.tsx

```diff
@@ -66,11 +67,23 @@ const AddServicePage = () => {
           }
         })
         .catch((err: AxiosError) => {
-          showErrorToast(
+          const errMessage = getErrorText(
             err,
             jsonData['api-error-messages']['create-ingestion-error']
           );
-          reject();
+          if (errMessage.includes('Failed to deploy Ingestion Pipeline')) {
```

Comment:
Nit : Can we move this message to constant `Failed to deploy Ingestion Pipeline`